theme-settings.php

```diff
@@ -132,7 +131,10 @@ function _nuboot_radix_file_set_permanent($fid) {
  * Implements hook_file_insert().
  */
 function nuboot_file_insert($file) {
+  // replace spaces with dash
   $file->filename = str_replace(' ', '-', $file->filename);
-  $hash = 'public://' . $file->filename;
-   file_move($file, $hash, 'FILE_EXIST_REPLACE');
+  // allow only letters and numbers and spaces
```

Comment:
Inline comments should start with a capital and end with a period: https://www.drupal.org/coding-standards/docs#inline